.travis.yml

```diff
@@ -1,17 +1,27 @@
 dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
```

Comment:
This line can be deleted because Xenial in now Travis’ default distro.

Reply:
Done in https://github.com/asciidoc/asciidoc-py3/pull/77